docs/apu_flashing_ipxe.md

```diff
@@ -0,0 +1,207 @@
+PC Engines APUx flashing with iPXE usage
+========================================
+
+Intro
+-----
+* Name of used APUx serial console visible in PC: `ttyUSB0`
+* IP of used PXE server: `192.168.0.108`
+
+Requirements
+------------
+
+* Ethernet connetion to the network
+* iPXE server have to be online in the same network
```

Comment:
`online and in the same network` or `to be in the same network`

Reply:
corrected